app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -153,22 +167,106 @@ describe('Week', () => {
     });
   });
 
-  test('should not display GlobalSpinner after loading data', async () => {
+  test('should display main content after data is loaded and hide GlobalSpinner', async () => {
```

Comment:
We need to confirm if we're using `test` or `it` with Shashi, because if we're going with `test`, we need to create a ticket to address all of the tests in the repository for consistency

Reply:
@choir27 100% agree! @shashilo ?